### ISSUE TITLE
Failed to load external module @babel/register on npm run build

### ISSUE THREAD

**Issue by dgmdevelopment** [wprig/wprig#221]
Hi, somewhat new to using gulp, node, npm.  I setup my local environment using ddev on win10 pc.  Thought i followed tutorial on wprig correctly, but on npm run build, i get an error and log report below.  Perhaps i did something incorrect, so this is likely a question, not a bug.  Thanks for helping me out!

```Failed to load external module @babel/register```

Log below - i removed my theme name.  

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'run', 'build' ]
2 info using npm@6.4.1
3 info using node@v11.2.0
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle wprig@1.0.4~prebuild: wprig@1.0.4
6 info lifecycle wprig@1.0.4~build: wprig@1.0.4
7 verbose lifecycle wprig@1.0.4~build: unsafe-perm in lifecycle true
8 verbose lifecycle wprig@1.0.4~build: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/var/www/html/web/wp/wp-content/themes/[redacted]/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
9 verbose lifecycle wprig@1.0.4~build: CWD: /var/www/html/web/wp/wp-content/themes/[redacted]
10 silly lifecycle wprig@1.0.4~build: Args: [ '-c', 'gulp' ]
11 silly lifecycle wprig@1.0.4~build: Returned: code: 1  signal: null
12 info lifecycle wprig@1.0.4~build: Failed to exec build script
13 verbose stack Error: wprig@1.0.4 build: `gulp`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
13 verbose stack     at EventEmitter.emit (events.js:182:13)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:182:13)
13 verbose stack     at maybeClose (internal/child_process.js:978:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:265:5)
14 verbose pkgid wprig@1.0.4
15 verbose cwd /var/www/html/web/wp/wp-content/themes/[redacted]
16 verbose Linux 4.9.125-linuxkit
17 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build"
18 verbose node v11.2.0
19 verbose npm  v6.4.1
20 error code ELIFECYCLE
21 error errno 1
22 error wprig@1.0.4 build: `gulp`
22 error Exit status 1
23 error Failed at the wprig@1.0.4 build script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]

**Comment by dgmdevelopment** [comment]
it was the 1.0.4. branch.. should i retry with 2.0?

**Comment by bamadesigner** [comment]
@dgmdevelopment can you try on `master` or `v2.0` branch?

**Comment by bamadesigner** [comment]
I just re-created on `1.0.4` and I get the same message.

@dgmdevelopment I think this issue has to do with conflicts between versions of babel and versions of npm. Ultimately, though, I don't think it has an effect on the compile itself and is just a warning.

Also, this issue has been addressed in WP Rig v2.0 which is about to be released.

With all that said, I wouldn't worry about it and the bug will be addressed when we release v2.0.

Thank you for reporting! Let me know if that doesn't make sense.
